src/content/apis/sports/free-nba.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Free NBA
+description: Unofficial NBA API for Historical NBA Data
+categories:
+  - sports
+features:
+  https: true
+url: https://www.balldontlie.io/#introduction
+logo: https://www.balldontlie.io/images/cryingjordan.jpeg

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.